pkg/tecdsa/common/common.go

```diff
@@ -0,0 +1,156 @@
+// Package common holds some common tools that can be used across multiple
+// tECDSA protocols, e.g. DKG and signing.
+package common
+
+import (
+	"fmt"
+	"github.com/bnb-chain/tss-lib/tss"
+	"github.com/keep-network/keep-core/pkg/crypto/ephemeral"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+	"math/big"
+	"strconv"
+)
+
+// GenerateTssPartiesIDs converts group member ID to parties ID suitable for
+// the TSS protocol execution.
+func GenerateTssPartiesIDs(
+	memberID group.MemberIndex,
+	groupMembersIDs []group.MemberIndex,
+) (*tss.PartyID, []*tss.PartyID) {
+	var partyID *tss.PartyID
+	groupPartiesIDs := make([]*tss.PartyID, len(groupMembersIDs))
+
+	for i, groupMemberID := range groupMembersIDs {
+		newPartyID := NewTssPartyIDFromMemberID(groupMemberID)
+
+		if memberID == groupMemberID {
+			partyID = newPartyID
+		}
+
+		groupPartiesIDs[i] = newPartyID
+	}
+
+	return partyID, groupPartiesIDs
+}
+
+// NewTssPartyIDFromMemberID creates a new instance of a TSS party ID using
+// the given member ID. Such a created party ID has an unset index since it
+// does not yet belong to a sorted parties IDs set.
+func NewTssPartyIDFromMemberID(memberID group.MemberIndex) *tss.PartyID {
```

Comment:
Should we export this function? It's used only from `common` package. Same question to the functions below.

Reply:
See https://github.com/keep-network/keep-core/pull/3228/commits/91a321da613ec37d61a1c236d195020737be0757